group/iab/IAB_Nomcom.md

```diff
@@ -54,17 +54,50 @@ So that the Nomcom is not later surprised, it is important to point them to the
-- In the event that a sitting IAB member was nominated to replace the casual vacancy in the IESG, the IETF Executive Director would be informed of the casual vacancy only upon confirmation of the initial nomination, and the Nominations Committee would then need to undertake a subsequent nominations process for the IAB casual vacancy.
+
+The following process will be used by the IAB for confirmation of nominated
+candidates for vacant IESG positions:
+
+- The IAB will undertake its best endeavour to make a confirmation decision
+  according to this process within the stipulated time period allowed for
+  confirmation, and not allow the decision process to lapse. If there are
+  extenuating circumstances that do not permit the IAB to make a decision
+  within the necessary timeframe, then these circumstances shall be addressed
+  as appropriate.
+
+- There shall be a positive vote for confirmation. The confirmation is
+  considered an IAB decision and follows the process documented in
+  [IAB_Decisions](IAB_Decisions.md). If that process does not lead to a
+  positive vote, the candidate is rejected.
+
+- Usually the confirmation of the entire slate is brought to an IAB vote as a
+  single decision. However, if the IAB fails to reach consensus on the entire
+  slate, the IAB can split the vote into multiple votes, each for a portion of
+  the slate, so that some of the candidates can be confirmed.
+
+- The IETF chair is disqualified from voting or participating in the discussion
+  on the IESG slate or IESG candidates. The IAB may ask the IETF chair
+  questions about the needs of the IESG prior to the IAB being advised of a
+  nominated candidate.
+
+- If any voting IAB member is a candidate for an IESG position, they must
+  recuse themselves and are disqualified from voting or participating in the
+  discussion for that specific confirmaiton. The vote is split to allow these
+  recused members to participate in the vote to confirm other IESG positions.
+
+- In the event that a sitting IAB member was nominated to replace the casual
+  vacancy in the IESG, the IETF Executive Director would be informed of the
+  casual vacancy only upon confirmation of the initial nomination, and the
+  Nominations Committee would then need to undertake a subsequent nominations
+  process for the IAB casual vacancy.
+
+- RFC 8713 does not mandate the criteria by which the IAB evaluates the IESG
+  slate for confirmation. The most common criterion is whether the IAB thinks
+  the NomCom process was followed correctly. However, other criteria are
+  possible; for example, a voting IAB member can decline to confirm a candidate
+  if they believe that the candidate's confirmation will be harmful to the
+  functioning of the IESG or IETF, or if they feel that the candidate's views
+  go against the IAB's technical direction for the Internet.
```

Comment:
I kinda agree, but doesn't this give an impression of IAB overreach (IETF needs to follow the technical direction from the IAB, whereas all we can do is advise).

Reply:
Yeah I think we could (and probably should) remove this example.

Reply:
We could just leave it at "other criteria are possible.."

Reply:
I've removed this bullet